lib/improver/tests/utilities/test_RegridLandSea.py

```diff
@@ -320,6 +320,35 @@ def test_with_regridding(self):
         self.assertDictEqual(result.attributes, self.cube.attributes)
         self.assertEqual(result.name(), self.cube.name())
 
+    @ManageWarnings(ignored_messages=["Using a non-tuple sequence for "],
+                    warning_types=[FutureWarning])
+    def test_multi_realization(self):
+        """Test that the expected changes occur and meta-data are unchanged
+        when handling a multi-realization cube."""
+        cube = self.cube.copy()
+        cube.coord('realization').points = [1]
+        cubes = iris.cube.CubeList([self.cube, cube])
+        cube = cubes.merge_cube()
```

Comment:
Is it possible to use the new test cube setup utilities here (and/or in setUp, but that might be out of scope)?  For example the `add_coordinate` function.

Reply:
As I was only copying an existing cube I didn't want to fiddle with the setup. The coordinate already existed in this case, I was just altering a value. Is that okay or do you want me to recreate the cubes with the setup function?

Reply:
Oh OK, sorry, I thought you were promoting a scalar coordinate.  In which case it would be nice to replace the cubes, but it's a faff so I wouldn't bother at this point.